doc/source/whatsnew/v0.24.0.txt

```diff
@@ -441,7 +441,7 @@ Datetimelike
 Timedelta
 ^^^^^^^^^
 
--
+- Fixed bug where array of timestamp and deltas raised a TypeError: unsupported operand type(s) for -: 'numpy.ndarray' and 'Timedelta' (:issue:`21980`)
```

Comment:
Take a look at how quotation marks and backticks are used elsewhere In this file (iOS keyboard doesn’t make it easy to give an example directly)

Reply:
I'll take a look!